README.md

```diff
@@ -151,6 +151,16 @@ After running `npm install`:
     // Use OktaSignIn
     var signIn = new OktaSignIn(/* configOptions */);
     ```
+3. [Browserify](http://browserify.org/) only: When you want to bundle your application with Browserify, you need to tell it that it doesn't have to parse the okta-signin-widget module. To do this, you need to add this flag when you run the `browserify` command:
+
+    ```
+    // Don't parse @okta/okta-signin-widget
+    --noparse=$PWD/node_modules/@okta/okta-signin-widget/dist/js/okta-sign-in.entry.js
+
+    // Example
+    browserify --noparse=$PWD/node_modules/@okta/okta-signin-widget/dist/js/okta-sign-in.entry.js main.js -o bundle.js
+    ```
+    The reason is that if you run the command without this flag, Browserify will attempt to resolve paths in `require()` calls in the already bundled okta-signin-widget, while it should ignore them since the `okta-sign-in.entry.js` is a standalone file and doesn't need the external modules, they are already bundled in the same file.
```

Comment:
Yeah, let's remove the reason. My feeling is that the README is more about how to do things rather than an in-depth explanation (especially in the case of how external libraries like browserify work) - if they're curious, I'd imagine they can either do a git blame to see when this commit was added (and see that it's linked to the issue where the reason was given) or just look up what no-parse does.